### DISCUSSION TITLE
Can we directly deploy Harbor through Docker Compose

### DISCUSSION THREAD

**Discussion by 984958198** [goharbor/harbor#19363] [Q&A]
Is there a mirror that includes all services in Harbor, so that it can be directly deployed and started through Docker Compose

**Comment by stonezdj** [comment]
Do you mean the online installer like that?
<img width="995" alt="Screenshot 2023-09-15 at 17 20 48" src="https://github.com/goharbor/harbor/assets/2696760/e36f29a2-6cb2-4159-abc1-59cdd2e86008">

**Reply by 984958198** [reply]
不是，我是指harbor所有服务在一个镜像里，现在是harbor包含7,8个镜像，不能由docker compose安装

**Reply by stonezdj** [reply]
目前不支持
